assets/js/blocks/cart-checkout/cart/full-cart/cart-line-item-row.js

```diff
@@ -20,28 +20,93 @@ import ProductLowStockBadge from './product-low-stock-badge';
 /**
  * Cart line item table row component.
  */
-const CartLineItemRow = ( { lineItem = {} } ) => {
+const CartLineItemRow = ( {
+	lineItem = {
```

Comment:
Should we define these default objects somewhere for reuse?

Reply:
Let's see how often it comes up?

Reply:
I ended up removing this and making `lineItem` (via propTypes). That way there's no need for defaults because it's clear via the propTypes that the properties are required.